pkgs/development/python-modules/grad-cam/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, matplotlib
+, numpy
+, opencv4
+, pillow
+, scikitlearn
+, torch
+, torchvision
+, ttach
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "grad-cam";
+  version = "1.4.6";
+  disabled = pythonOlder "3.6";
+  format = "setuptools";
```

Comment:
Let's use pyproject here instead. It has a better idea about the separation between build and runtime dependencies.

Might need to add `setuptools` into `nativeBuildInputs` then.

Reply:
Sure, done.